### PR TITLE
resolve issue#49 with Dragen-v3.7 only inputs

### PR DESCRIPTION
**Pull request by skanwal**
Attempt to resolve https://github.com/umccr/RNAsum/issues/49.

It'd be great if you can test it with your set-up of different input combinations (Dragen-v3.9/Arriba, Dragen-v3.7/Arriba, Dragen-v*/withoutArriba, bcbio) to make sure the code changes work for your various requirements.

### CODE REVIEW

**Comment by JMarzec** [comment]
I've updated the code and checked all 12 possible combinations of Dragen 3.5.7 / Dragen 3.9.3 /  bcbio VS Arriba / no Arriba / WTS + WGS / WTS only.

Main changes:
- Set Dragen v3.9.3 as default input
- Set umccrise results for example run as default
- Overall, clean the fusions filtering step
- Define chunks processing Dragen + Arriba, Arriba / Arriba + Pizzly and Pizzly only
- Distinguish chunks processing Dragen's fusion format version 3.9.3 or prior to version 3.9.3
- Clean fusions summary table

File: rmd_files/RNAseq_report.Rmd [hunk 1]
```diff
@@ -3619,21 +3622,31 @@ if ( runArribaChunk ) {
 ##### Otherwise add empty columns expected from Aribba results
```
Comment (skanwal) [inline rmd_files/RNAseq_report.Rmd:3622]: Should we remove this comment as we are no longer adding empty columns for Arriba results?

File: rmd_files/RNAseq_report.Rmd [hunk 2]
```diff
@@ -3755,24 +3768,18 @@ if ( runArribaChunk ) {
...
+      fusions <- rbind(fusions, data.frame(geneA=pizzly.fusions$geneA[i],geneB=pizzly.fusions$geneB[i], breakpointA="-", breakpointB="-", siteA="-", siteB="-", type="-", split_reads=pizzly.fusions$split_reads[i], split_readsA="-", split_readsB="-", discordant_mates=pizzly.fusions$discordant_mates[i], confidence="-", FGID=pizzly.fusions$FGID[i], reported_fusion=pizzly.fusions$reported_fusion[i], reported_fusion_geneA=pizzly.fusions$reported_fusion_geneA[i], reported_fusion_geneB=pizzly.fusions$reported_fusion_geneB[i], effector_gene=pizzly.fusions$effector_gene[i], fusions_cancer=pizzly.fusions$fusions_cancer[i], geneA_dna_support="-", geneB_dna_support="-", Arriba="-", Pizzly="Yes" ))
     }
   }
 ##### Otherwise add empty columns expected from Aribba results
```
Comment (skanwal) [inline rmd_files/RNAseq_report.Rmd:3774]: Same as before i.e. removing/updating this comment?

File: rmd_files/RNAseq_report.Rmd (hunk 1 above)
Comment (JMarzec) [reply rmd_files/RNAseq_report.Rmd]: Good point

**Pull request merged**